documentation/README.md

```diff
@@ -31,7 +31,7 @@ project.ext.documentation = [
 apply from: '../config/documentation/doclava/android.gradle'
 ```
 
-## Dokka
+## For Dokka
```

Comment:
Maybe we should move the Dokka instructions above Doclava? Since Kotlin is the default used these days. What do you think?

Reply:
Moved Dokka above Doclava